models/aws_project.rb

```diff
@@ -43,7 +43,7 @@ def add_sdk_objects
     @pricing_checker = Aws::Pricing::Client.new(access_key_id: self.access_key_ident, secret_access_key: self.key)
   end
 
-  def daily_report(date=(Date.today - 2), slack=true, text=true, rerun=false, verbose=false)
+  def daily_report(date=(DEFAULT_DATE), slack=true, text=true, rerun=false, verbose=false)
```

Comment:
I'm not sure we need to specify a default for `date` here, as every time we call `daily_report` we are specifying a value for date.

Reply:
It is a bit redundant in our current implementation, but with the costs shenanigans it's pretty vital, so inclined to leave it in so it's there for any (hypothetical) future use of these methods.